src/components/Forms/AddProductForm.tsx

```diff
@@ -253,22 +249,41 @@ const AddEditProductForm = () => {
 					))}
 				</div>
 				<div className="h-[10%] grid grid-cols-1 ipad:grid-cols-2 gap-3 items-center">
-					<Button
-						buttonType="submit"
-						url={null}
-						title={
-							loading ? (
-								<>
-									<IconLoader className="animate-spin mr-1" />{' '}
-									{'processing....'}
-								</>
-							) : (
-								'Save'
-							)
-						}
-						color="bg-action-success"
-						otherStyles="rounded-md h-10"
-					/>
+					{productData.id ? (
+						<Button
+							buttonType="submit"
+							url={null}
+							title={
+								processing ? (
+									<>
+										<IconLoader className="animate-spin mr-1" />{' '}
+										{'processing....'}
+									</>
+								) : (
+									'Save'
```

Comment:
I was expecting this button to have title of update. You can fix this, then we're good.
<img width="1440" alt="Screenshot 2024-07-08 at 16 02 00" src="https://github.com/atlp-rwanda/hackers-ec-Fe/assets/151399071/6b021ad4-a8e8-4f4e-b36d-8a256625846b">